tools/mascot/src/main/java/dk/alexandra/fresco/tools/mascot/field/FieldElementSerializer.java

```diff
@@ -0,0 +1,90 @@
+package dk.alexandra.fresco.tools.mascot.field;
+
+import dk.alexandra.fresco.framework.network.serializers.ByteSerializer;
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class FieldElementSerializer implements ByteSerializer<FieldElement> {
+
+  private final BigInteger modulus;
+  private final int modBitLength;
+  private final FieldElementUtils fieldElementUtils;
+
+  /**
+   * Creates new {@link FieldElementSerializer}.
+   * 
+   * @param modulus modulus of field elements
+   * @param modBitLength mod bit length
+   */
+  public FieldElementSerializer(BigInteger modulus, int modBitLength) {
+    this.modulus = modulus;
+    this.modBitLength = modBitLength;
```

Comment:
As for the FieldElement class: It should not be necessary  to give `modBitLength` as a parameter. It should be possible to derive from the modulus it self.

Reply:
Done.